examples/extensions-zalando/README.md

```diff
@@ -1,5 +1,5 @@
-# Swagger Plugin Zalando Extension
-Zalando's architecture centers around microservices, and each API is documented with a Swagger specification. Zalando has its own [API Guidelines](https://opensource.zalando.com/restful-api-guidelines) that extends the standard Swagger specification. This extension example illustrates how the Swagger Plugin can be extended to support Zalando specific guidelines for REST APIs.
+# Zalando OpenAPI Editor with Zalando Extensions
```

Comment:
Is this an example extension for the plugin or what is actually meant by this directory?

Reply:
> Is this an example extension for the plugin or what is actually meant by this directory?

It's both an actual plugin and an example of extension :) https://plugins.jetbrains.com/plugin/11115-swagger-zalando-extensions- The most recent release is from 2019 though..